qr-code-auth/README.md

```diff
@@ -0,0 +1,286 @@
+# Collection - QrCodeAuth
+
+## API Endpoints
+
+|                                       Route                                        |                                         Description                                          |
+| :--------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------: |
+|                      [POST /qr-code-auth](#post-qr-code-auth)                      |                            Creates a new authentication document.                            |
+|                       [GET /qr-code-auth](#get-qr-code-auth)                       |           Retrieves all the authentication document belonging to a specified user            |
+| [PATCH /qr-code-auth](#patch-qr-code-authuser_iduser_idis_authorizedis_authorized) | Updates the is_authorized field of an existing qr-code-auth document for the specified user. |
+
+## POST /qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
```

Comment:
@shreya-mishra , could you please tell when this call will be made? Will it be from mobile to backend or web app to backend?
1. In first case, is_authorized and access_token don't make sense.
2. In second case, access_token does not make sense.
Could you please clarify this further?

Reply:
updated it , POST call request body should only contain userId and device_info